### PR TITLE
bundle images, styles and fonts via webpack

### PR DESCRIPTION
**Pull request by simonmarklar**
- note - I dont have a dev environment so I cannot test but the built source for both production and dev seems to be ok

### CODE REVIEW

**Comment by hawken93** [comment]
I believe it needs quotes here:
```css
.fresh {
    background-image: url(../img/fresh.svg);
}

.rotten {
    background-image: url(../img/rotten.svg);
}
```
It does not get picked up currently.

File: webpack.config.js [hunk 1]
```diff
@@ -1,22 +1,26 @@
...
+        extensions: ['.ts', '.js']
     },
     plugins: [
         new CleanWebpackPlugin(),
+        new CopyPlugin({
+            patterns: [{ from: 'favicon.ico', to: '.' }]
+        }),
         new HtmlWebpackPlugin({
             filename: 'index.html',
             template: 'index.html',
```
Comment (hawken93) [inline webpack.config.js:26]: add `favicon: 'favicon.ico'` here

**Comment by hawken93** [comment]
Cool! I'll recompile. So far LGTM.

**Comment by hawken93** [comment]
It messes up svg :(
```
cat da29ab07bb9c0eadef387b72fd731f4b.svg
export default __webpack_public_path__ + "cb6e840e08726299bf8f968fe08d7ead.svg";
```

**Comment by simonmarklar** [comment]
are you sure that its borked and not just how webpack handles svgs ? do they not display in the app ? (wish i could check)

it appears every SVG has 2 files, with one being JS exporting the other. 

i've got a few projects with webpack 4 using svgs, so i'll investigate further after work

**Comment by hawken93** [comment]
Svg are an xml format, it didn't load correctly as webpack handled it like js

**Comment by simonmarklar** [comment]
i've not had a chance to look yet, got a real life issue that is taking all my attention. Should be able to get to this in a couple of days

**Comment by simonmarklar** [comment]
oh dear, i just noticed in your code snippet, im including SVG in the webpack build twice. that will teach me for using copy/pasta!

fixed that, done a dev and prod build and the svg files are no longer doubling up with one being a js file.

**Comment by hawken93** [comment]
I've tested this and it looks good!

I just wish husky wouldn't run and obscure all the actual changes but that's not your fault.

**Review by hawken93 — APPROVED** [review]

**Comment by simonmarklar** [comment]
> I've tested this and it looks good!
> 
> I just wish husky wouldn't run and obscure all the actual changes but that's not your fault.

awesome!

husky only runs on the staged files, once we've got all the outstanding PRs merged in we can run prettier on the entire codebase and we should be golden. IMHO its worth the initial pain to get a consistent coding style but i do feel your pain :)

**Review by YouKnowBlom — APPROVED** [review]

**Pull request merged**